### PR TITLE
Add StackItem ToJson

### PR DESCRIPTION
**Pull request by bettybao1209**
close https://github.com/neo-project/neo-devpack-dotnet/issues/218  https://github.com/neo-project/neo-node/issues/560

### CODE REVIEW

File: neo-cli/CLI/Helper.cs [hunk 1]
```diff
@@ -19,5 +29,78 @@ public static bool IsYes(this string input)
 
             return input == "yes" || input == "y";
         }
+        public static JObject ToJson(this StackItem item)
+        {
+            return ToJson(item, null);
+        }
+
+        private static JObject ToJson(StackItem item, List<(StackItem, JObject)> context = null)
```
Comment (Tommo-L) [inline neo-cli/CLI/Helper.cs:37]:
```suggestion
        private static JObject ToJson(StackItem item)
```

We only need this method.

**Comment by Tommo-L** [comment]
For me, it's ok. And if we can move `ToJson` to vm, it's benifit for neo-cli, rpc module, etc.

**Comment by erikzhang** [comment]
Please use https://github.com/neo-project/neo/pull/1569

**Comment by Tommo-L** [comment]
The same too, https://github.com/neo-project/neo-modules/pull/221#issuecomment-614361806

**Comment by erikzhang** [comment]
Use `Neo.3.0.0-CI00892` directly.

File: neo-cli/CLI/Helper.cs [hunk 2]
```diff
@@ -1,3 +1,12 @@
+using Neo.IO.Json;
+using Neo.VM.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Array = Neo.VM.Types.Array;
+using Boolean = Neo.VM.Types.Boolean;
+using Buffer = Neo.VM.Types.Buffer;
+
```
Comment (erikzhang) [inline neo-cli/CLI/Helper.cs:9]: This file should be reverted.

**Pull request merged**